working/wildcards/feature-specification.md

```diff
@@ -181,7 +181,7 @@ main() {
 ```
 
 Import prefixes named `_` are non-binding and will provide access to the
```

Comment:
I'd say "but" instead of "and" here. Maybe even "but will still"

The part after "and" is how non-`_` prefixes work too, so erasing it should make no difference.
It's there for emphasis, saying that *even though* `_` does not introduce a name for the imports, the extensions still work.